systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMakerST.java

```diff
@@ -663,6 +666,36 @@ void testScaleMirrorMakerSubresource() {
             assertThat(pod.contains(mmGenName), is(true));
         }
     }
+
+    @Test
+    void testScaleMirrorMakerToZero() {
+        // Deploy source kafka
+        KafkaResource.kafkaEphemeral(kafkaClusterSourceName, 1, 1).done();
+        // Deploy target kafka
+        KafkaResource.kafkaEphemeral(kafkaClusterTargetName, 1, 1).done();
+
+        KafkaMirrorMakerResource.kafkaMirrorMaker(CLUSTER_NAME, kafkaClusterTargetName, kafkaClusterSourceName, "my-group" + rng.nextInt(Integer.MAX_VALUE), 3, false)
+            .editMetadata()
+                .addToLabels("type", "kafka-mirror-maker")
+            .endMetadata()
+            .done();
+
+        String mmDepName = KafkaMirrorMakerResources.deploymentName(CLUSTER_NAME);
+        List<String> mmPods = kubeClient().listPodNames("type", "kafka-mirror-maker");
+        assertThat(mmPods.size(), is(3));
+
+        LOGGER.info("Scaling MirrorMaker to zero");
+        KafkaMirrorMakerResource.replaceMirrorMakerResource(CLUSTER_NAME, mm -> mm.getSpec().setReplicas(0));
+
+        KafkaMirrorMakerUtils.waitForKafkaMirrorMakerReady(CLUSTER_NAME);
+        PodUtils.waitForPodsReady(kubeClient().getDeploymentSelectors(mmDepName), 0, true);
+
+        mmPods = kubeClient().listPodNames("type", "kafka-mirror-maker");
+        KafkaMirrorMakerStatus mmStatus = KafkaMirrorMakerResource.kafkaMirrorMakerClient().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getStatus();
+
+        assertThat(mmPods.size(), is(0));
+        assertThat(mmStatus.getConditions().get(0).getType(), is("Ready"));
```

Comment:
We maybe should take into consideration to create some Enum on CR type statuses. Would be better instead of hardcoding these values

Reply:
I have it on the list, as many things, so I will do it in another PR as it's not related just to this test